tests/SpecTests/RetryableReadsSpecTest.php

```diff
@@ -1,135 +0,0 @@
-<?php
-
-namespace MongoDB\Tests\SpecTests;
-
-use stdClass;
-
-use function basename;
-use function file_get_contents;
-use function glob;
-use function is_object;
-use function strpos;
-
-/**
- * Retryable reads spec tests.
- *
- * @see https://github.com/mongodb/specifications/tree/master/source/retryable-reads
- * @group serverless
- */
-class RetryableReadsSpecTest extends FunctionalTestCase
```

Comment:
Note: PHP has yet to implement any of the retryable reads prose tests. The CMAP prose test doesn't apply, and the mongos-related tests are waiting on [PHPC-1911](https://jira.mongodb.org/browse/PHPC-1911).

As such, there was nothing else in this file that needed to be kept.